README.md

```diff
@@ -59,6 +59,122 @@ true
 false
 ```
 
+Remainder:
```

Comment:
I realise that the readme already had examples in, but I think examples should go into the doc-comments instead, so that they can be seen on Pursuit.

Reply:
I agree about examples in the doc-comments. I'd be happy to completely rework the README to give a shorter and more general overview of the functions Data.Number supports but that will mean changing what was already there before this PR.

Reply:
That sounds good to me, thanks!

Reply:
After some delay (my apologies), this should now be resolved in #626e299